internal/kubernetes/lock.go

```diff
@@ -13,13 +13,29 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
-func TryAcquireLock(ctx context.Context, client client.Client, lockName types.NamespacedName, owner metav1.Object) error {
```

Comment:
Is it intended that variable is named `errLogInUse`  and not `errLockInUse`? 

I also would rather just have the err msg as constant and then create the error in the method. But also fine declaring the err here.

Reply:
This comment is about this line 

```
var errLogInUse = errors.New("lock is already acquired by other resources")
```